Builds/containers/packaging/dpkg/debian/control

```diff
@@ -0,0 +1,21 @@
+Source: rippled
+Section: misc
+Priority: extra
+Maintainer: Ripple Inc. <support@ripple.com>
```

Comment:
The company is NOT called "Ripple Inc.".

This wrong name is used in various repositories and even in the description of the organization on Github...

Reply:
@MarkusTeufelberger I can change the pkg files to "Ripple Labs, Inc.", but the other items will have to be addressed by someone else. thanks.

Reply:
Sure, thanks. You probably have better ways to know/reach the respective repository owners anyways. :-)